test/tests/carousel-1.js

```diff
@@ -35,14 +36,9 @@ ariaTest('Carousel 1: section has aria-roledescription set to carousel', example
   await assertAttributeValues(t, ex.landmarkSelector, 'aria-roledescription', 'carousel');
 });
 
-ariaTest('Carousel 1: previous and next buttons have role button', exampleFile, 'carousel-button-role-link', async (t) => {
+ariaTest('Carousel 1: pause, previous and next buttons have aria-label', exampleFile, 'carousel-button-aria-label', async (t) => {
```

Comment:
The names of the tests are printed like this: `carousel/carousel-1/carousel-1.html [data-test-id="carousel-button-aria-controls"]: Carousel 1: previous and next buttons have aria-controls Rejected promise returned by test`

So `Carousel 1:` can be removed from the beginning of each test name in this file, as the example file being tested is prepended to the name.

Reply:
we will eventually have another carousel example, but it is not going to make this release, but I guess it will have its own test file, so I think it could be removed to reduce the verbosity of the messages.